escoba/src/registerServiceWorker.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-console */
+
+import { register } from 'register-service-worker'
+
+if (process.env.NODE_ENV === 'production') {
```

Comment:
can you explain which aspect of the project this class is a part of ?

Reply:
it was boiler plate from vue cli.  i think maybe has to do with pwa option.  not sure.